portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,4 +88,16 @@ private String convertToJsonUsingGson() {
     Gson gson = new Gson();
     return gson.toJson(comments);
   }
+
+  /**
+   * Converts an string to an integer.
+   */
+  private int toInt(String str) {
+    try {
+      return Integer.parseInt(str);
+    } catch (NumberFormatException e) {
+      // Return 3 as a default because no selection has been made.
+      return 3;
```

Comment:
It seems unclear why we would want to return 3 as a default here. Can you clarify that?